doc/release_notes/v0.3.rst

```diff
@@ -85,7 +85,7 @@ New features
 - updated namespace name to reflect directory name
 - made mock fac/inst/region/market classes to be used with testing. added the stub facility in a new stub namespace and related tests.
 - added initial stub directory and adjusted cmake files to include their tests.
-- moved all dynamic loading into xml_file_loader. Added a method for listing installed/discoverable dynamic modules to env class. added rng schema methods to test agents and removed rng files from them. removed rng installation from cmake module macro. added master and module schema dumping to cyclus binary. added schema agent test (that schema parses). moved heavy stuff out of xml-file-loader constructor. renamed LoadAll to LoadSim.
+- moved all dynamic loading into xml_file_loader. Added a method for listing installed/discoverable dynamic modules to env class. added rng schema methods to test agents and removed rng files from them. removed rng installation from cmake module macro. added main and module schema dumping to cyclus binary. added schema agent test (that schema parses). moved heavy stuff out of xml-file-loader constructor. renamed LoadAll to LoadSim.
```

Comment:
Same comment about the schema vocabulary

Reply:
ditto